WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -623,6 +629,13 @@ extension ProductFormViewModel {
         }
     }
 
+    func hasChangesExcludingImageUploads() -> Bool {
```

Comment:
For the moment I see this is used just here.

```suggestion
    private func hasChangesExcludingImageUploads() -> Bool {
```

Reply:
Moved this method to a private extension in f0af7d8410263464798b25278406cc7a2a4ce88e.